CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-rc.1 December 2, 2021
+
+## Features ✨ and improvements 🏁
+* Fix suggestion message to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```

Comment:
move to bugs, it's not a feature
`Features ✨ and improvements 🏁` section is not needed